### PR TITLE
Next zowex is zowe

### LINKED ISSUE #938: [next] Color escape characters not parsed by daemon
The `zowex` binary produces output that looks like this:
```
←[33m DESCRIPTION←[39m
←[33m -----------←[39m

   Manage z/OS jobs.

←[33m USAGE←[39m
←[33m -----←[39m

   zowe zos-jobs <group>

   Where <group> is one of the following:

←[33m GROUPS←[39m
←[33m ------←[39m

   cancel | can  Cancel a job
```

**Environment**
* OS: Windows 10 v2004
* Terminal (daemon/CLI): Powershell-Powershell or Bash-Bash

### LINKED ISSUE #978: unable to escape and pass quoted parameters in zowex
OK got that up and running, thx.
Got an issue with zowex now see hereafter, do I open new issue or continue in this one?

With zowe:
fbeyl@pvslt03 ~ $ iplrest=$(zowe zos-tso send address-space $sko --data "ex (panel) 'PARM'")
fbeyl@pvslt03 ~ $ echo $iplrest
[ { "DSNAME":"VENDOR.PARMLIB" , "SEQ":"1" , "VOLSER":"VPMVSD" , "BLKSIZE":"6160" , "EXTENT":"1" , "SMS":"NO" , "LRECL":"80" , "DSORG":"PO" , "RECFM":"FB" , "CRDATE":"2003.119" , "REFDATE":"2021.071" , "SYSNAME":"S0W1" , "SYSLEVEL":"z/OS 02.03.00 HBB77B0" } , { "DSNAME":"SVTSC.PARMLIB" , "SEQ":"2" , "VOLSER":"VTMVSG" , "BLKSIZE":"6160" , "EXTENT":"1" , "SMS":"NO" , "LRECL":"80" , "DSORG":"PO" , "RECFM":"FB" , "CRDATE":"1997.118" , "REFDATE":"2017.300" , "SYSNAME":"S0W1" , "SYSLEVEL":"z/OS 02.03.00 HBB77B0" } , { "DSNAME":"LVL0.PARMLIB" , "SEQ":"3" , "VOLSER":"VTLVL0" , "BLKSIZE":"6160" , "EXTENT":"7" , "SMS":"NO" , "LRECL":"80" , "DSORG":"PO" , "RECFM":"FB" , "CRDATE":"2007.025" , "REFDATE":"2021.035" , "SYSNAME":"S0W1" , "SYSLEVEL":"z/OS 02.03.00 HBB77B0" } , { "DSNAME":"SYS1.PARMLIB" , "SEQ":"4" , "VOLSER":"VIMVSB" , "BLKSIZE":"27920" , "EXTENT":"1" , "SMS":"NO" , "LRECL":"80" , "DSORG":"PO" , "RECFM":"FB" , "CRDATE":"2017.116" , "REFDATE":"2021.035" , "SYSNAME":"S0W1" , "SYSLEVEL":"z/OS 02.03.00 HBB77B0" } ] READY

With zowex:
fbeyl@pvslt03 ~ $ iplrest=$(zowex zos-tso send address-space $sko --data "ex (panel) 'PARM'")
Command Error:
Onbekende argumenten: (panel), 'PARM'
Command failed due to improper syntax
Command entered: "zowe zos-tso send address-space Z40275-137-aackaaah --data ex (panel) 'PARM' --dcd /home/fbeyl/"
Available commands are "address-space".
Use "zowe zos-tso send --help" to view groups, commands, and options.
Error Details:
Onbekende argumenten: (panel), 'PARM'

Tried escaping but till now no luck.

Zowex should leave " in the command because it has to be sent to tso session.

_Originally posted by @fbeyl in https://github.com/zowe/zowe-cli/issues/974#issuecomment-797893428_

### PR DESCRIPTION
**Pull request by gejohnston**
This PR enables 'zowex' to be named 'zowe'. This executable can:
- Automatically start the daemon behind the scenes.
- Eliminates the visible window associated with the daemon.
- An environment variable named ZOWE_USE_DAEMON can be set to "no" to prevent the use of the daemon. The command is then passed to the traditional zowe-CLI command.
    - Thus, you can temporarily use the traditional Zowe CLI command to correct some display limitations (like displaying colors).

The majority of the changes associated with this PR (not merged in from other PRs) are contained in main.rs

This PR also fixes the following issues:
- resolves  #938
- resolves #978

This PR should NOT be merged until:

- The following imperative PR is merged into next and its corresponding (but not yet created) zowe-cli changes are also merged into next.

    zowe/imperative#688

- The following PR is merged into this PR:

    #1194

### CODE REVIEW

**Review by zFernand0 — DISMISSED** [review]
LGTM! 😋 
Thanks for documenting the outcome of the different approaches tried when starting the daemon.

File: zowex/src/main.rs [hunk 1]
```diff
@@ -261,6 +404,212 @@ fn get_beg(buf: &str) -> Vec<String> {
...
+ */
+fn user_wants_daemon() -> bool {
+    const DAEMON_ENV_VAR_NM: &str = "ZOWE_USE_DAEMON";
+    let env_var_val;
+    match env::var(DAEMON_ENV_VAR_NM) {
+        Ok(val) => env_var_val = val,
+        Err(_e) => env_var_val = "NoDaemon".to_string(),
+    }
+
+    if env_var_val.to_lowercase() == "false" || env_var_val.to_lowercase() == "no" {
```
Comment (t1m0thyj) [inline zowex/src/main.rs:504]: Should we also support `0` as a falsy value? I don't think any change is required here, just curious 🙂 I'm not sure what recommended practices are for allowed values to specify a boolean flag in an environment variable.

**Review by t1m0thyj — DISMISSED** [review]

File: zowex/src/main.rs (hunk 1 above)
Comment (gejohnston) [reply zowex/src/main.rs]: When we doc this, I suggest that we only document "no". I added the "false" option for those who refuse to read doc. In that spirit, adding 0 is also reasonable. I added that third option.

**Comment by zFernand0** [comment]
PR LGTM!
Will approve after merge conflicts are resolved : )

**Review by zFernand0 — DISMISSED** [review]

**Review by t1m0thyj — DISMISSED** [review]

**Review by dkelosky — APPROVED** [review]

**Pull request merged**